Makefile

```diff
@@ -2,7 +2,8 @@
 TEST_INFRA_VERSION ?= latest
 
 # Version of gRPC core used for the gRPC driver
-DRIVER_VERSION ?= v1.45.0
+# Pinned to include https://github.com/grpc/grpc/pull/29160
```

Comment:
If this is the only pr needed, we probably wan to pin to that particular commit? It is also okay to just keep like this if we would like to update this very soon in the next several PRs.

Reply:
I picked a newer commit to include other changes needed for our release 1.2.0, such as https://github.com/grpc/grpc/pull/29207, https://github.com/grpc/grpc/pull/29197 and https://github.com/grpc/grpc/pull/29057. I will update the comment to remove the specific 
reason for pinning the driver, we can also figure something out next Monday.

Reply:
I have updated the commit to just include what is necessary for v1.2.0

Reply:
For future reference, if we are making a change, we should explain what we are doing. If we are updating to include three things, we should mention all three things and then choose the newest of the three commits to pin on. Also, one reason why we are doing this is because these changes are necessary to support the new functionality in this release, and grpc/grpc release 1.46.0 is not out yet.